tests/system/robot/chromeTests.py

```diff
@@ -2473,3 +2473,48 @@ def test_i13307():
 		]),
 		message="jumping into region with aria-labelledby should speak label",
 	)
+
+
+def test_textParagraphNavigation():
+	_chrome.prepareChrome("""
+		<!-- First a bunch of paragraphs that don't match text regex -->
+		<p>Header</p>
+		<p>Liberal MP: 1904–1908</p>
+		<p>.</p>
+		<p>!</p>
+		<p>?</p>
+		<p>…</p>
+		<p>5.</p>
+		<p>test....</p>
+		<p>a.b</p>
+		<p></p>
+		<!-- Now a bunch of matching paragraphs -->
+		<p>Hello, world!</p>
+		<p>He replied, "That's wonderful."</p>
+		<p>He replied, "That's wonderful".</p>
+		<p>He replied, "That's wonderful."[4]</p>
+		<p>Предложение по-русски.</p>
+		<p>我不会说中文！</p>
+		<p>Bye-bye, world!</p>
+	""")
+
+	expectedParagraphs = [
+		'''Hello, world!''',
+		'''He replied,  That's wonderful.''',
+		'''He replied,  That's wonderful .''',
+		'''He replied,  That's wonderful.  4''',
+		'''Предложение по-русски.''',
+		'''我不会说中文！''',

```

Comment:
Could you also add comments to each of these e.g.

```suggestion
		# Tests regex condition for CJK full width character terminators
		"我不会说中文！",
```